Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
The execline build also generates a static library; `--disable-static` disables building this library.
I don't know why I explicitly asked it to not build when submitting the formula, but it's a prerequisite for `s6`.